.github/workflows/trigger-dag-reuse-wf.yaml

```diff
@@ -34,17 +29,45 @@ on:  # yamllint disable-line rule:truthy
       astronomer_key_secret:
         description: 'astro cloud astronomer_key_secret'
         required: true
+      organization_id:
+        description: 'astro cloud organization_id'
+        required: true
+      bearer_token:
```

Comment:
Do we know if this Bearer token has an expiry and if yes, how frequently we need to update this?

Reply:
@pankajkoti i have set Expiry to None for now